WalletWasabi.Gui/Global.cs

```diff
@@ -172,10 +172,19 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 
 				#region BitcoinStoreInitialization
 
-				await bstoreInitTask.ConfigureAwait(false);
+				try
+				{
+					await bstoreInitTask.ConfigureAwait(false);
 
-				// Make sure that the height of the wallets will not be better than the current height of the filters.
-				WalletManager.SetMaxBestHeight(BitcoinStore.IndexStore.SmartHeaderChain.TipHeight);
+					// Make sure that the height of the wallets will not be better than the current height of the filters.
+					WalletManager.SetMaxBestHeight(BitcoinStore.IndexStore.SmartHeaderChain.TipHeight);
+				}
+				catch (Exception ex)
+				{
+					//WalletManager.SetMaxBestHeight(0);
```

Comment:
This is the trick (obviously it should be uncommented.) Also it could be the block height of the first bech32 filter, we must have a constants somewhere about it.